pkgs/by-name/ra/rabbit/package.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "rabbit";
+  version = "1.0.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "natarajan-chidambaram";
+    repo = "RABBIT";
+    rev = version;
+    hash = "sha256-IRG3OcWutkZA4oegeEIDyaIadl3dLaMneqOt/H2/Il4=";
+  };
+
+  pythonRelaxDeps = true;
+
+  nativeBuildInputs = [
+    python3.pkgs.setuptools
+    python3.pkgs.wheel
+    python3.pkgs.pythonRelaxDepsHook
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
```

I am not a Python expert, but nowadays this is written as
```suggestion
  dependencies = with python3.pkgs; [
```

---

Ho first time I see this. Curious to learn more about it!

---

Done, thank you, TIL !